package.json

```
@@ -13,6 +13,7 @@
     "react-scripts": "5.0.1",
     "redux": "^4.2.1",
     "sass": "^1.60.0",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
```

Could you please add engines here to ensure in version you are working with.
```suggestion
  },
  "engines": {
    "node": "18.x",
    "npm": "9.3.x",
    "yarn": "1.22.x"
  },
```